hw/top_earlgrey/cdc/cdc_waivers.misc.tcl

```diff
@@ -4,76 +4,99 @@
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_rv_dm.dap.i_dmi_jtag_tap.tap_state_q[3:0]") && (Signal == "IOR0") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_rxf_ptr_wptr.q[12:0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.wptr[12:0]") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+set_rule_status -rule {W_DATA} -expression {(ReceivingFlop == "top_earlgrey.u_spi_device.u_reg.u_txf_ptr_rptr.q[12:0]") && (Signal == "top_earlgrey.u_spi_device.u_fwmode.u_txf_ctrl.rptr[12:0]") && (Association == "None")} -status {Waived}  -comment {included in waived paths}
+
+
+# W_MASYNC : included in waived paths
+set_rule_status -rule {W_MASYNC} -expression {(ReceivingFlop =~ "top_earlgrey.u_spi_device.u_rxf_overflow.src_level") && (MultiClockDomains =~ "IO_DIV4_CLK::SPI_DEV_IN_CLK,SPI_DEV_PASSTHRU_IN_CLK") && (Driver =~ "top_earlgrey.u_spi_device.u_reg.u_control_mode.q[0]")} -status {Waived} -comment {static info in IO_DIV4_CLK}
+set_rule_status -rule {W_MASYNC} -expression {(ReceivingFlop =~ "top_earlgrey.u_spi_device.u_readcmd.u_readsram.u_fifo.gen_normal_fifo.u_fifo_cnt.*ptr_o*") && (MultiClockDomains =~ "IO_DIV2_CLK,IO_DIV4_CLK,SPI_DEV_IN_CLK,SPI_DEV_PASSTHRU_IN_CLK::SPI_DEV_IN_CLK,SPI_DEV_PASSTHRU_IN_CLK") && (Driver =~ "top_earlgrey.u_spi_device.u_memory_2p.b_rvalid_sram_q")} -status {Waived} -comment {static info in IO_DIV4_CLK}
+set_rule_status -rule {W_MASYNC} -expression {(ReceivingFlop =~ "top_earlgrey.u_spi_device.u_p2s.out_shift[7:0]") && (MultiClockDomains =~ "IO_DIV2_CLK,IO_DIV4_CLK,SPI_DEV_IN_CLK,SPI_DEV_PASSTHRU_IN_CLK::SPI_DEV_OUT_CLK,SPI_DEV_PASSTHRU_OUT_CLK") && (Driver =~ "top_earlgrey.u_spi_device.u_fwmode.u_tx_fifo.storage[7:0][7:0]")} -status {Waived} -comment {reg data is muxed with spi data in spi clock domain}
+set_rule_status -rule {W_MASYNC} -expression {(ReceivingFlop =~ "top_earlgrey.u_spi_device.u_p2s.out_shift[7:0]") && (MultiClockDomains =~ "IO_DIV4_CLK::SPI_DEV_OUT_CLK,SPI_DEV_PASSTHRU_OUT_CLK") && (Driver =~ "top_earlgrey.u_spi_device.u_reg.u_cfg_tx_order.q[0]")} -status {Waived -comment {reg data is muxed with spi data in spi clock domain}}
+set_rule_status -rule {W_MASYNC} -expression {(ReceivingFlop =~ "top_earlgrey.u_spi_device.u_fwmode.u_rxf_ctrl.sram_wdata[31:0]") && (MultiClockDomains =~ "SPI_DEV_IN_CLK,SPI_DEV_PASSTHRU_IN_CLK::IO_DIV2_CLK,IO_DIV4_CLK,SPI_DEV_IN_CLK,SPI_DEV_PASSTHRU_IN_CLK") && (Driver =~ "top_earlgrey.u_spi_device.u_fwmode.u_rx_fifo.storage[7:0][7:0]")} -status {Waived -comment {reg data is muxed with spi data in spi clock domain}}
```

Comment:
here and in the waiver above: is the syntax correct? it looks like the braces are not closed after {Wavied ...

Reply:
Ah. Good catch!

Reply:
Fixed

Reply:
Looks like these are still here (maybe due to the rebase exercise).

Reply:
Thanks for double check. It's fixed